.gitignore

```diff
@@ -20,3 +20,7 @@ build/
 
 # OS X
 .DS_Store
+
+## Ros2 Incompleted Modules
+jupyros/ros2/turtle_sim.py
+notebooks/ROS2_Turtlesim.ipynb
```

Comment:
We usually do not add files from local branches here.

If there is something that you do not want to commit, you can select what to commit by staging specific files. For example `git add jupyros/ros2/publisher.py` instead of adding everything.